dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/CollectionsController.java

```diff
@@ -283,8 +286,16 @@ public String view(
     Collection collection = service.findOne(uuid);
     List<Locale> existingLanguages =
         languageSortingHelper.sortLanguages(displayLocale, collection.getLabel().getLocales());
+    Set<Locale> existingSubcollectionLanguages =
+        collection.getChildren().stream()
+            .map(child -> child.getLabel().getLocales())
+            .flatMap(l -> l.stream())
+            .collect(Collectors.toSet());
+    existingSubcollectionLanguages =
```

Comment:
Warum wird hier sortiert wenn's danach direkt wieder in ein `HashSet` (das unsortiert ist) geworfen wird?

Reply:
Die Entfernung von Duplikaten wird jetzt direkt in Sortierungshelfer erledigt.